protocol-impl/src/main/java/io/zeebe/protocol/impl/record/value/message/WorkflowInstanceSubscriptionRecord.java

```diff
@@ -140,4 +151,8 @@ public WorkflowInstanceSubscriptionRecord setCloseOnCorrelate(final boolean clos
     closeOnCorrelateProp.setValue(closeOnCorrelate);
     return this;
   }
+
+  public DirectBuffer getCorrelationKeyBuffer() {
```

Comment:
Ignore this getter for JSON serialization.

```suggestion
  @JsonIgnore
  public DirectBuffer getCorrelationKeyBuffer() {
```